Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/.Documentation/BytesInDevicePortal.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: f5d74b997f52d224ea01b4b80ffd492e
+TextureImporter:
```

Comment:
Can also get rid of the `.meta` files for these images, since they won't be loaded in Unity and aren't needed.

Reply:
Removed unnecessary .meta files.